.gitignore

```diff
@@ -0,0 +1,4 @@
+pyproject.toml
```

Comment:
You don't want to ignore [pyproject.toml](https://packaging.python.org/en/latest/specifications/pyproject-toml/). It's the most important configuration file in your whole project (aside from Docker Compose YAML).